apps/user-service/src/auth/dto/verification-user-response.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { JsonApiAttributes } from "@terramatch-microservices/common/dto/json-api-attributes";
+import { ApiProperty } from "@nestjs/swagger";
+
+@JsonApiDto({ type: "verifications", id: "uuid" })
```

Comment:
🎨 I'll go ahead and approve, but `uuid` is the default and doesn't need to be explicitly specified.